src/libs/actions/Policy.js

```diff
@@ -30,7 +75,69 @@ function getPolicySummaries() {
         });
 }
 
+/**
+ * Fetches the policyList from the API and saves a simplified version in Onyx
+ */
+function getPolicyList() {
+    GetPolicyList()
+        .then((data) => {
+            if (data.jsonCode === 200) {
+                const policyDataToStore = _.reduce(data.policyList, (memo, policy) => ({
+                    ...memo,
+                    [`${ONYXKEYS.COLLECTION.POLICY}${policy.id}`]: getSimplifiedMemberList(policy),
+                }), {});
+                Onyx.mergeCollection(ONYXKEYS.COLLECTION.POLICY, policyDataToStore);
+            }
+        });
+}
+
+/**
+ * Merges the passed in login into the specified policy
+ *
+ * @param {String} login
+ * @param {String} welcomeNote
+ * @param {String} policyID
+ */
+function invite(login, welcomeNote, policyID) {
+    const key = `${ONYXKEYS.COLLECTION.POLICY}${policyID}`;
+
+    // Make a shallow copy to preserve original data, and concat the login
+    const policy = _.clone(allPolicies[key]);
+    policy.employeeList = [...policy.employeeList, login];
```

Comment:
No, it's a shallow copy, you'll be mutating the original `employeeList` 

```js
let original = { subArray: [1, 2] }
let cloned = _.clone(original);
cloned.subArray.push(3);

console.log(original.subArray) // [1, 2, 3]
```

Reassigning with `[...policy.employeeList, login]` won't mutate original data

Reply:
Yep I am mistaken

Reply:
Ok cool, I'll revert that, even if it doesn't matter in this case I'd rather avoid having somebody do that somewhere else where it could cause problems.